detox/src/configuration/index.test.js

```diff
@@ -39,7 +39,7 @@ describe('composeDetoxConfig', () => {
         argv: {
```

Comment:
As a general note (not directly related to the PR), this test is a bit too integrative to my taste. With the various configuration domain each broken down to its own composer (session, runner, apps, etc) along with the awesome unit testing coverage -- I feel a total mocking of each with some minimal glue-logic testing here should suffice.

I know the approach is arguable, no need to accept it. In any case, no need to address this right now.

Reply:
Let's add a TODO to discuss this later.